server/src/bots/telegram/botResponse/startResponse.ts

```diff
@@ -1,9 +1,33 @@
-import {getFullMenuKeyboard} from '../services/keyboard';
-import {greetUser} from '../../../messages/userMessage';
-import {showHelpInfoResponse} from './helpResponse';
-import {CallBackQueryHandler} from '../models';
+import { getFullMenuKeyboard } from '../services/keyboard';
+import { greetUser } from '../../../messages/userMessage';
+import { showHelpInfoResponse } from './helpResponse';
+import { CallBackQueryHandler } from '../models';
+import { getUser, addUser } from '../../../services/domain/user'
+import { logger } from '../../../utils/logger';
+import {User} from '../../../models/user.model'
 
 export const startResponse: CallBackQueryHandler = async (bot, message, chatId) => {
+    let user = await getUser(chatId);
```

Comment:
- type `let user: User = await getUser(chatId);`
- let's not force a user to wait for that. You can give a user the response and, meanwhile, write him in Firebase

Reply:
2. fixed